static/js/Misc.jsx

```diff
@@ -1164,18 +1180,21 @@ const ReorderEditorWrapper = ({toggle, type, data}) => {
     is a dictionary of the topic whose sources can be accessed via its `refs` field.
      */
     const reorderingSources = data.length !== 0;
-    const filterAndSortRefs = (refs) => {
+    const _filterAndSortRefs = (refs) => {
+        if (!refs) {   
+            return [];
```

Comment:
based on the code `_filterAndSortRefs(data.refs?.about?.refs)` a ref can be inputted as undefined so the check should happen on the function